README.md

```diff
@@ -46,110 +40,87 @@ Once we have the path set up, we can try to run R. Don't worry if you aren't an
-	100 job(s) submitted to cluster 14027.
+	Submitting job(s)..........
+	1 job(s) submitted to cluster 3796250.
 	$ condor_q user
 	 
-	-- Submitter: login01.osgconnect.net : <128.135.158.173:47839> : login01.osgconnect.net
-	 ID      OWNER            SUBMITTED     RUN_TIME ST PRI SIZE CMD
-	14027.0   user           8/25 22:51   0+00:00:43 R  0   0.0  R-wrapper.sh mcpi.
-	14027.1   user           8/25 22:51   0+00:00:43 R  0   0.0  R-wrapper.sh mcpi.
-	14027.2   user           8/25 22:51   0+00:00:31 R  0   0.0  R-wrapper.sh mcpi.
-	14027.4   user           8/25 22:51   0+00:00:41 R  0   0.0  R-wrapper.sh mcpi.
-	14027.5   user           8/25 22:51   0+00:00:41 R  0   0.0  R-wrapper.sh mcpi.
+	$ condor_q
+	-- Schedd: login03.osgconnect.net : <192.170.227.22:9618?... @ 05/13/19 09:51:04
+	OWNER      BATCH_NAME     SUBMITTED   DONE   RUN    IDLE  TOTAL JOB_IDS
+	user	   ID: 3796250   5/13 09:50      _      _      1      1 3796250.0
 	...
 
-You can follow the status of your job cluster with the `watch condor_q` command, which shows `condor_q` output that refreshes each 2 seconds.  Press `control-C` to stop watching.
-
-Since our jobs just output their results to standard out, we can do the final analysis from the log files. Let's see what one looks like:
-
-	$ cat log/mcpi.out.14027.1
-	[1] 3.141246
+You can follow the status of your job cluster with the `connect watch` command, which shows `condor_q` output that refreshes each 5 seconds.  Press `control-C` to stop watching.
 
-After job completion we have 100 Monte Carlo estimates of the value of pi. Taking an average across them all should give us a closer approximation.
+Since our jobs prints to standard out, we can check the output files. Let's see what one looks like:
 
-We'll use a bit of awk magic to do the averaging:
-
-	$ grep "[1]" log/mcpi.out.* | awk '{sum += $2} END { print "Average =", sum/NR}'
-	Average = 3.14151
-
-That's pretty close! With even more sample sets — that is, more Queue jobs in the cluster — we can statistically come even closer.
-
-##What to do next?
-The `R.submit` file may have included a few lines that you are unfamiliar with.  For example, `$(Cluster)` and `$(Process)` are variables that will be replaced with the job's cluster and process id.  This is useful when you have many jobs submitted in the same file.  Each output and error file will be in a separate directory.
-
-Also, did you notice the transfer_input_files line?  This tells HTCondor what files to transfer with the job to the worker node.  You don't have to tell it to transfer the executable, HTCondor is smart enough to know that the job will need that.  But any extra files, such as our MonteCarlo R file, will need to be explicitly listed to be transferred with the job.  You can use transfer_input_files for input data to the job, as shown in [Transferring data with HTCondor](https://github.com/OSGConnect/tutorial-htcondor_transfer).
+	$ cat R.out.3796250.0
+	[1] "Hello World!"
 
 ## Getting Help
```

Comment:
Maybe add links to the other guides -- submitting multiple jobs + using libraries?

Reply:
👍Added links to other R guides!